packages/iris-grid/src/sidebar/visibility-ordering-builder/sortable-tree/SortableTree.tsx

```diff
@@ -0,0 +1,350 @@
+  );
+  const sensors = useSensors(
+    useSensor(PointerWithInteraction, {
+      activationConstraint: {
+        distance: 5,
+      },
+    })
+    // useSensor(KeyboardSensor, {
+    //   coordinateGetter,
+    // })
+  );
+
+  const sortedIds = useMemo(() => flattenedItems.map(({ id }) => id), [
+    flattenedItems,
+  ]);
+  const activeItem =
+    activeId != null ? flattenedItems.find(({ id }) => id === activeId) : null;
+
+  useEffect(() => {
+    sensorContext.current = {
+      items: flattenedItems,
+      offset: offsetLeft,
+    };
+  }, [flattenedItems, offsetLeft]);
+
+  return (
+    <DndContext
+      sensors={sensors}
+      collisionDetection={closestCenter}
+      measuring={measuring}
+      onDragStart={handleDragStart}
+      onDragMove={handleDragMove}
+      onDragOver={handleDragOver}
+      onDragEnd={handleDragEnd}
+      onDragCancel={handleDragCancel}
+    >
+      <SortableContext items={sortedIds} strategy={verticalListSortingStrategy}>
+        <div className={classNames(activeId != null && styles.MarchingAnts)}>
+          {flattenedItems.map(item => {
+            const { id, children, collapsed, depth } = item;
+            return (
+              <SortableTreeItem
+                key={id}
+                id={id}
+                value={id}
+                depth={id === activeId && projected ? projected.depth : depth}
+                indentationWidth={indentationWidth}
+                indicator={indicator}
+                collapsed={Boolean(collapsed && children.length)}
+                onCollapse={
+                  collapsible && children.length
+                    ? () => handleCollapse(id)
+                    : undefined
+                }
+                onRemove={
+                  removable && onRemove ? () => handleRemove(id) : undefined
+                }
+                item={item}
+                renderItem={renderItem}
+              />
+            );
+          })}
+        </div>
+        {createPortal(
+          <DragOverlay
+            dropAnimation={dropAnimationConfig}
+            modifiers={indicator ? [adjustTranslate] : undefined}
+          >
+            {activeId && activeItem ? (
+              <SortableTreeItem
+                id={activeId}
+                depth={activeItem.depth}
+                clone
+                childCount={getChildCount(items, activeId) + 1}
+                value={activeId.toString()}
+                indentationWidth={indentationWidth}
+                renderItem={renderItem}
+                item={activeItem}
+              />
+            ) : null}
+          </DragOverlay>,
+          document.body
+        )}
+      </SortableContext>
+    </DndContext>
+  );
+
+  function handleDragStart({ active: { id: newActiveId } }: DragStartEvent) {
+    setActiveId(newActiveId);
+    setOverId(newActiveId);
+
+    document.body.style.setProperty('cursor', 'grabbing');
```

Comment:
Not sure if there's any differences between this and doing it on `document.documentElement` like `Grid` does. In any case, this works.

Reply:
Probably not unless you render content outside the body